site/user-guide.md

```diff
@@ -66,7 +68,7 @@ You can have one or more operators in your Kubernetes cluster that manage one or
 We provide a Helm chart to manage the installation and configuration of the operator.
 Detailed instructions are available [here](install.md).
 
-## Exposing applications outside the Kubernetes cluster
+### Exposing applications outside the Kubernetes cluster
```

Comment:
I am not sure if the changes to the level of heading here and below are correct. We need to make sure they match what is in the table of contents section at least.

Reply:
You're right. I will move both Exposing applications outside the Kubernetes cluster and Operator Docker images down a level in the TOC. Thanks.